pkg/identities/handlers.go

```diff
@@ -0,0 +1,108 @@
+package identities
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/canonical/identity-platform-admin-ui/internal/http/types"
+	"github.com/canonical/identity-platform-admin-ui/internal/logging"
+	"github.com/go-chi/chi/v5"
+	kClient "github.com/ory/kratos-client-go"
+)
+
+type API struct {
+	service ServiceInterface
+	baseURL string
+
+	logger logging.LoggerInterface
+}
+
+func (a *API) RegisterEndpoints(mux *chi.Mux) {
+	mux.Get("/api/v0/identities", a.handleList)
+	mux.Get("/api/v0/identities/{id}", a.handleDetail)
+	// mux.Post("/api/v0/identities", a.handleCreate)
+	// mux.Put("/api/v0/identities/{id}", a.handleUpdate)
+	// mux.Patch("/api/v0/identities/{id}", a.handlePartialUpdate)
+	// mux.Delete("/api/v0/identities/{id}", a.handleRemove)
+	// mux.Delete("/api/v0/identities/{id}/sessions", a.handleSessionRemove)
+	// mux.Delete("/api/v0/identities/{id}/credentials/{type}", a.handleCrededntialRemove)
+}
+
+func (a *API) handleList(w http.ResponseWriter, r *http.Request) {
+	pagination := types.ParsePagination(r.URL.Query())
+
+	credID := r.URL.Query().Get("credID")
+
+	ids, err := a.service.ListIdentities(r.Context(), pagination.Page, pagination.Size, credID)
+
+	if err != nil {
```

Comment:
Have a question here. It seems that we construct the error responses and responses differently in different package, e.g. `pkg/clients/handler.go`. Do we need to somehow unify it?

This also leads to another question about the framework option. Do you think it's necessary we pick up the `https://github.com/gin-gonic`?

Reply:
we will bring this together under one type

re `gin-gonic` that is a more fully fledged framework whereas `go-chi` is simply a router, see it as the difference between flask and django

we were using `gorilla/mux` in the login ui so best move was to `go-chi` but happy to reevaluate